scripts/changed_components.rb

```diff
@@ -14,8 +14,7 @@ def tsort_each_child(node, &block)
 end
 
 def get_hab_deps(source_dir)
-  plan_path = "#{source_dir}/habitat/plan.sh"
-  depString = `bash -c '. #{plan_path}; echo ${pkg_deps[*]}'`
+  depString = `bash -c 'cd ${source_dir}; PLAN_CONTEXT=habitat . habitat/plan.sh; echo ${pkg_deps[*]}'`
```

Comment:
⚠️ Pre Coffee Review: Should this be `#{source_dir}`?

Reply:
Shoot I'll fix that...